crates/hyperdrive-wasm/src/long.rs

```diff
@@ -66,3 +66,85 @@ pub fn getMaxLong(
 
     U256::from(result_fp).to_string()
 }
+
+/// Gets the curve fee paid by longs for a given base amount.
+///
+/// @param poolInfo - The current state of the pool
+///
+/// @param poolConfig - The pool's configuration
+///
+/// @param baseAmount - The amount of base tokens to spend
+#[wasm_bindgen(skip_jsdoc)]
+pub fn getOpenLongCurveFees(
+    poolInfo: &JsPoolInfo,
+    poolConfig: &JsPoolConfig,
+    baseAmount: String,
+) -> String {
+    set_panic_hook();
+    let state = State {
+        info: poolInfo.into(),
+        config: poolConfig.into(),
+    };
+
+    let _baseAmount: FixedPoint = FixedPoint::from(U256::from_dec_str(&baseAmount).unwrap());
```

Comment:
Nit: Some of the existing functions have these type annotations on them, but they're unnecessary. The type will be inferred.
```suggestion
    let _baseAmount = FixedPoint::from(U256::from_dec_str(&baseAmount).unwrap());
```

Reply:
Awesome, good to know!